R/terminal.R

```diff
@@ -0,0 +1,270 @@
+
+#' Send Text to a Terminal
+#'
+#' Send text to an existing terminal.
+#'
+#' @param id The terminal id. The \code{id} is obtained from
```

Comment:
For documentation of parameters that are reused across multiple places, you can share documentation between functions with a couple of mechanisms:

The first option is through the use of `roxygen` templates; we use those extensively in `sparklyr`. See e.g.

https://github.com/rstudio/sparklyr/blob/master/man-roxygen/roxlate-ml-features.R
https://github.com/rstudio/sparklyr/blob/master/R/ml_utils.R#L30

(the use of `roxlate-` as a prefix in the template file name is not required; I just adopted that as a convention since it plays well with RStudio's fuzzy finder (`Ctrl + .`))

The second option is to document it in a single function, and then have other functions inherit the parameter documentation from that function, e.g.

```
@inheritParams terminalList
```

See e.g.


https://github.com/tidyverse/dplyr/blob/fa36b50f343e7dde4e5c316b4c864489ccc0f2e2/R/colwise-arrange.R#L7
https://github.com/tidyverse/dplyr/blob/fa36b50f343e7dde4e5c316b4c864489ccc0f2e2/R/colwise.R#L53

Note that the documentation there is tied to `NULL`, but the `@name` roxygen attribute is set so that it can be used for inheritance.

I don't think it's necessary to adopt one of these schemes for this PR but just wanted you to know what infrastructure is available for this sort of thing.